File: course-materials/week16-cross-language-samples/cpp/shared_example.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string name;
    cout << "Name: ";
    cin >> name;

    for (int i = 1; i <= 3; i++) {
        cout << "Hello, " << name << "! Round " << i << endl;
    }

    return 0;
}

File: course-materials/week16-cross-language-samples/java/SharedExample.java
import java.util.Scanner;

public class SharedExample {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Name: ");
        String name = scanner.nextLine();

        for (int i = 1; i <= 3; i++) {
            System.out.println("Hello, " + name + "! Round " + i);
        }

        scanner.close();
    }
}
